server/portal/apps/accounts/managers/ssh_keys.py

```diff
@@ -59,7 +59,7 @@ def _tacc_prompt_handler(
         """
         answers = {
             'password': self.password,
-            'tacc_token_code': self.token
+            'tacc_token': self.token
```

Comment:
for me its wanting `tacc_token_code` for frontera

Reply:
Ah so interesting. It was prompting me for `tacc_code`, and `tacc_token_code` was causing it to crash. Hmm

Reply:
yes, for `frontera`.  testing `cloud.data.community` now so I'm curious what it prompts for.

Reply:
it looks like you can just add both to prompt answers:
```        answers = {
            'password': self.password,
            'tacc_token_code': self.token,
            'tacc_token': self.token
        }
```

Reply:
Ah, perfect